torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -2833,7 +2833,6 @@ def test_return_future(self):
                 return_future,
             )
 
-    @_skip_if_tensorpipe_agent
     @dist_init
```

Comment:
Has this been stress-tested? Just checking to ensure we don't introduce more flaky TensorPipe agent tests

Reply:
Ran it with 500 stresstests and they all passed.